apps/designer/app/designer/features/style-panel/controls/text/text-control.tsx

```diff
@@ -0,0 +1,228 @@
+import { ChangeEvent, FocusEvent, KeyboardEvent, useRef } from "react";
+import {
+  Box,
+  Text,
+  Grid,
+  IconButton,
+  TextField,
+  Tooltip,
+  IconButtonWithMenu,
+  numericScrubControl,
+} from "@webstudio-is/design-system";
+import { getFinalValue } from "../../shared/get-final-value";
+import { useIsFromCurrentBreakpoint } from "../../shared/use-is-from-current-breakpoint";
+import { ControlProps } from "../../style-sections";
+import { iconConfigs, StyleConfig } from "../../shared/configs";
+import { type StyleValue, type Unit, units } from "@webstudio-is/react-sdk";
+import { ChevronDownIcon } from "@webstudio-is/icons";
+
+const CSSGridLeft = {
+  gridArea: "1 / 1 / 2 / 2",
+};
+
+const CSSGridRight = {
+  gridArea: "-1 / -1 / -2 / -2",
+};
+
+const CSSIconButton = {
+  zIndex: 1,
+  borderRadius: 2,
+  height: "calc($sizes$6 - 6px)",
+  width: "calc($sizes$6 - 6px)",
+  transform: "translate(3px, 3px)",
+  "&:focus": {
+    boxShadow: "none",
+  },
+};
+
+const CSSTextUnit = {
+  fontSize: "$1",
```

Comment:
you should not need to set the fontSize manually, $1 is 12 px in design system, also I just realized that I used px values for some reason in the Text component, so I don't know what went through my head when I did this (I used the values straight from figma) but they should have been $1 $2 etc from the theme

Reply:
I will change those px values to theme values in a bit directly on main, but we should not need to set this here in any case